README.md

```diff
@@ -125,16 +141,37 @@ module load java matlab conda2
 # Get the latest version of the pipeline
 nextflow pull labsyspharm/mcmicro-nf
 
-# All previous commands require an additional `-profile O2` flag
-nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2
-nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --tma -profile O2
+# All previous commands require an additional `-profile O2` flag and must be run from a compute node
+srun -p priority -t 0-12 --mem 8G nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2
+srun -p priority -t 0-12 --mem 8G nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --tma -profile O2
+```
+
+In the above, `-t 0-12 --mem 8G` requests 12 hours of compute time and 8GB of memory from the O2 cluster. To avoid running over on your disk quota, it is also recommended to use `/n/scratch2` for holding the `work/` directory:
```

Comment:
Mention that scratch2 is also faster than /home or /n/groups so jobs will complete faster.